### PR TITLE
Service Validator: "DateTime" property in odimra and plugin manager is not the valid date format - Bug fix

### PR DESCRIPTION
**Pull request by AngithCJ**
**Changes**
- DateTime property format changed to as expected for odimra and plugin manager

### CODE REVIEW

**Review by irappa-pattar — APPROVED** [review]

**Comment by shivacharanms** [comment]
Add UT changes for the same.

**Review by amar-shalgar — APPROVED** [review]

**Review by irappa-pattar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Review by hemkapoor — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

File: svc-managers/managers/managers_test.go [hunk 1]
```diff
@@ -68,7 +69,7 @@ func TestGetManager(t *testing.T) {
 	assert.Equal(t, "Service", manager.ManagerType, "Status code should be StatusOK.")
 	assert.Equal(t, req.ManagerID, manager.ID, "Status code should be StatusOK.")
 	assert.Equal(t, "1.0", manager.FirmwareVersion, "Status code should be StatusOK.")
```
Comment (irappa-pattar) [inline svc-managers/managers/managers_test.go:71]: Could you please change the message in all assert.Equal
Ex. assert.Equal(t, "1.0", manager.FirmwareVersion, "**Firmware version should be 1.0**")

**Review by irappa-pattar — CHANGES_REQUESTED** [review]

File: svc-managers/managers/managers_test.go (hunk 1 above)
Comment (AngithCJ) [reply svc-managers/managers/managers_test.go]: I have updated the PR. Please review?

**Review by irappa-pattar — APPROVED** [review]
LGTM

**Pull request merged**